english/movie-ripoffs/hogwarts/hogwarts.md

```diff
@@ -25,4 +25,6 @@ from other students as the elderly wizard leaps up the staircase in misguided pu
 
 "Don't forget to put on your invisible cloak, Sir" calls Caroline as he disappears out of sight.  
 
-"We just tell him that he's invisible when he's wearing it - he's soooo gullible." giggles Caroline, rolling her eyes.
\ No newline at end of file
+"We just tell him that he's invisible when he's wearing it - he's soooo gullible." giggles Caroline, rolling her eyes.
+
+Just then a redheaded kid with a rat in one hand and broken wand in the other walks up to you. He puts the rat in his pocket, sticks his hand out to you and says, "Hi I'm Rorry Weasley."
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.